clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -599,12 +605,14 @@ def create_participants_df_AIBL(input_path, clinical_spec_path, clinical_data_di
         This methods create a participants file for the AIBL dataset where
         information regarding the patients are reported
 
-        :param input_path: path to the input directory :param
-        clinical_spec_path: path to the clinical file :param clinical_data_dir:
-        directory to the clinical data files :param delete_non_bids_info: if
-        True delete all the rows of the subjects that are not available in the
-        BIDS dataset :return: a pandas dataframe that contains the participants
-        data and it is saved in a tsv file
+        :param input_path: path to the input directory
+        :param clinical_spec_path: path to the clinical file
+        :param clinical_data_dir: directory to the clinical data files
+        :param delete_non_bids_info: if True delete all the rows of the subjects
+        that are not available in the BIDS dataset
+
+        :return:
+            a pandas DataFrame that contains the participants data and it is saved in a tsv file
```

Comment:
Since you modified this docstring, could you take this opportunity to format with Google Python Style? :)

Reply:
Done in https://github.com/aramis-lab/clinica/pull/144/commits/c16f67984ebf321ce46ba553f1b7f8a631557555